sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -51,8 +51,8 @@ def _from_secret_item(cls, secret_item):
         # type: (_models.SecretItem) -> SecretProperties
         """Construct a SecretProperties from an autorest-generated SecretItem"""
         return cls(
-            secret_item.attributes,
-            secret_item.id,
+            secret_item.attributes,  # type: ignore
+            secret_item.id,  # type: ignore
             content_type=secret_item.content_type,
```

Comment:
It looks like `content_type`, `tags`, and possibly other properties should technically be optional as well

Reply:
Here is ignoring the errors that expect `attributes` and `id` to be **optional** while we don't want to set them with `None`.

Reply:
True -- I just mean that I'm surprised mypy doesn't complain despite the `content_type` property on line 62, for example, not being listed as possibly `None`